auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java

```diff
@@ -98,29 +107,44 @@ public void unbindService() {
      *
      * @param uri the uri to open in a Custom Tab or Browser.
      */
-    public void launchUri(@NonNull final Uri uri) {
+    public void launchUri(@NonNull final Uri uri, @NonNull final boolean launchAsTwa) {
```

Comment:
Is this a breaking change, as it's a public method that now requires two non-null parameters?

Reply:
This shouldn't be an issue as CustomTabsController is a package private class and `launchUri` is not exposed outside the library.

I removed the @NonNull annotation as we are using primitive and it doesn't need that annotation